src/main/java/io/interop/finsemble/example/fdc3/JavaFDC3Example.java

```diff
@@ -0,0 +1,327 @@
+        final JTextField broadcastTickerTextField = new JTextField("MSFT");
+        // The user channels are only available AFTER Finsemble has connected; use an onConnected handler
+        onConnectedHandlers.add(() -> {
+            for (final String userChannel:getUserChannels()) {
+                broadcastChannelComboBox.addItem(userChannel);
+            }
+        });
+        // The button to raise the intent
+        final JButton broadcastButton = new JButton("Broadcast");
+        // Add an action for when the intent button is clicked.
+        broadcastButton.addActionListener(event -> {
+            if (null != broadcastChannelComboBox.getSelectedItem()) {
+                broadcast(
+                    broadcastChannelComboBox.getSelectedItem().toString(),
+                    broadcastTickerTextField.getText().trim()
+                );
+            }
+        });
+        // GridBagConstraints
+        final GridBagConstraints cs = new GridBagConstraints();
+        cs.fill = GridBagConstraints.HORIZONTAL;
+        cs.gridy = 0;
+        // Combobox
+        cs.gridx = 0;
+        cs.anchor = GridBagConstraints.WEST;
+        cs.gridwidth = GridBagConstraints.RELATIVE;
+        cs.weightx = 0;
+        broadcastPanel.add(broadcastChannelComboBox, cs);
+        // Ticker text field
+        cs.gridx = 1;
+        cs.gridwidth = GridBagConstraints.REMAINDER;
+        cs.weightx = 100;
+        broadcastPanel.add(broadcastTickerTextField, cs);
+        // Button
+        cs.gridx = 0;
+        cs.gridy = 1;
+        broadcastPanel.add(broadcastButton, cs);
+        // Add the "broadcast" panel to the tabbed panel
+        tabbedPane.add("Broadcast", broadcastPanel);
+
+
+        // Layout and add the components
+        getContentPane().setLayout(new BorderLayout());
+        getContentPane().add(tabbedPane, BorderLayout.CENTER);
+        // Pack the components
+        pack();
+        // Set the window size
+        setSize(600, 125);
+        // Center the window
+        setLocationRelativeTo(null);
+        // Display the window
+        setVisible(true);
+    }
+
+    /**
+     * Executes any onConnected handlers for Finsemble.
+     */
+    private void executeOnConnectedHandlers() {
+        // Safely handle each handler
+        onConnectedHandlers.forEach(IgnoreException::ignoreException);
+    }
+
+
+    /**
+     * The main point of entry.
+     *
+     * @param args command line arguments
+     * @throws Exception when something goes wrong
+     */
+    public static void main(final String[] args) throws Exception {
+        // Set the look and feel (it is OK to throw an exception here)
+        ignoreException(() -> UIManager.setLookAndFeel(UIManager.getSystemLookAndFeelClassName()));
+
+        // Open a new UI instance
+        new JavaFDC3Example(args);
+    }
+
+}
+
+//<editor-fold defaultstate="collapsed" desc="apps.json App Definition">
+/* This is a sample app config suitable for insertion into apps.json
+ {
```

Comment:
The `name` property is deprecated, isn't it?

Is the `interop` property needed, since the app doesn't listen for any intents?

The `windowType` is redundant since `type` is native, isn't it?

Does `addToWorkspace` need to be false?